shared/modules/authentication/config.ts

```diff
@@ -0,0 +1,27 @@
+import { Env } from '@metamask/profile-sync-controller/sdk';
+import { ENVIRONMENT } from '../../../development/build/constants';
+
+/**
+ * Check if the force auth match build is enabled.
+ *
+ * @returns true if the force auth match build is enabled, false otherwise
+ */
+export function isForceAuthMatchBuild() {
+  return process.env.FORCE_AUTH_MATCH_BUILD?.toString() === 'true';
+}
+
+export function isDevEnvironment() {
+  return process.env.METAMASK_ENVIRONMENT === ENVIRONMENT.DEVELOPMENT;
+}
+
+export function loadAuthenticationConfig(): Env {
```

Comment:
Hmm might need to doublecheck this func - It only returns PRD and UAT - I don't see DEV? Unless that is the intention.